packages/hoppscotch-common/src/modules/i18n.ts

```diff
@@ -123,11 +123,6 @@ export default <HoppModule>{
       fallbackLocale: "en",
       legacy: false,
       allowComposition: true,
```

Comment:
can you also remove this unused import 

`import en from "../../locales/en.json"`